server/design/keys/design.go

```diff
@@ -73,6 +73,28 @@ var _ = Service("keys", func() {
 		Meta("openapi:extension:x-speakeasy-name-override", "revokeById")
 		Meta("openapi:extension:x-speakeasy-react-hook", `{"name": "RevokeAPIKey"}`)
 	})
+
+	Method("verifyKey", func() {
+		Description("Verify an api key")
+
+		Security(security.ByKey)
```

Comment:
Just checking does this work without us defining a scope. 
Scope("producer")

Like doesn't it work for both consumer and producer scoped keys locally